test_debugger.py

```
@@ -0,0 +1,17 @@
+# flake8-debugger (T10) - catches debugger statements
+def test_debugger():
+    # T100: Use of debugger
+    import pdb; pdb.set_trace()  # T100
```

Ruff (0.12.2)

Multiple statements on one line (semicolon)

(E702)